src/content.ts

```diff
@@ -2093,7 +2093,7 @@ async function voteAsync(type: number, UUID: SegmentUUID, category?: Category):
     const sponsorIndex = utils.getSponsorIndexFromUUID(sponsorTimes, UUID);
 
     // Don't vote for preview sponsors
-    if (sponsorIndex == -1 || sponsorTimes[sponsorIndex].source !== SponsorSourceType.Server) return;
+    if (sponsorIndex == -1 || sponsorTimes[sponsorIndex].source !== SponsorSourceType.Server) Promise.resolve();
```

Comment:
Will this still short-circuit the function?

Reply:
well it's that or Promise.reject()

Reply:
Shouldn't there still be a return though after resolving the promise. Otherwise it's going to run all the code underneath

Reply:
oh oops